django/contrib/staticfiles/runserver_nostatic/management/commands/runserver.py

```diff
@@ -0,0 +1,8 @@
+"""
+This file exists solely to shadow the `runserver` command provided by
```

Comment:
You can put `django.contrib.staticfiles.runserver_nostatic` above `django.contrib.staticfiles` in INSTALLED_APPS to recover the original runserver command and use WhiteNoise during development. This is only needed because of deprecated runserver command in staticfiles app and can be removed together later. See **5. Using WhiteNoise in development**:
http://whitenoise.evans.io/en/stable/django.html